coordinator/src/services/validation/bundle-validator.ts

```diff
@@ -126,6 +127,15 @@ export function verifyPrescriptionBundle(bundle: fhir.Bundle): Array<errors.Vali
         "https://fhir.nhs.uk/R4/StructureDefinition/Extension-DM-ResponsiblePractitioner",
         "MedicationRequest.extension"
       )
+    ),
+    (medicationRequests: Array<fhir.MedicationRequest>) => verifyValueIdenticalForAllMedicationRequests(
```

Comment:
is it worth adding a test to `request-validator.spec.ts` around this being enforced?

Reply:
also i know you've only added one of the validators here but there has to be a more concise way of creating this array

Reply:
Didn't bother adding a test as we already have tests for the consistency check, and the only difference here is which field is being considered. As for creating the array, Conner's neatened it up on branch AEA-628-improve-request-validator.